HyprMXAdapter/src/main/java/com/chartboost/mediation/hyprmxadapter/HyprMXAdapter.kt

```diff
@@ -620,6 +633,8 @@ class HyprMXAdapter : PartnerAdapter {
         listener: PartnerAdListener?,
     ): Result<PartnerAd> {
         PartnerLogController.log(SHOW_STARTED)
+        loadedPartnerPlacements.remove(partnerAd.request.partnerPlacement)
```

Comment:
you sure you can load an ad as soon as you show? And not after the ad gets dismissed?

Reply:
Good question.... Sounds like putting this in dismiss would be the best course of action.  I'll do that.

Reply:
Done.  I dev tested it and this works fine too.  Definitely the safest approach.